app/docs/0.11.x/configuration.md

```diff
@@ -694,6 +694,11 @@ Kong will resolve hostnames as either `SRV` or `A` records (in that order, and
 In case a name was resolved as an `SRV` record it will also override any given
 port number by the `port` field contents received from the dns server.
 
+The dns options `SEARCH` and `NDOTS` (from the `/etc/resolv.conf` file) will
+be used to expand short names to fully qualified ones. So it will first try the
+entire `SEARCH` list for the `SRV` type, if that fails it will try the `SEARCH`
+list for `A`, etc.
```

Comment:
This does feel very colloquial as well (especially the "So" at the beginning of the sentence).